src/Bicep.Core.IntegrationTests/Emit/ParamsFileWriterTests.cs

```diff
@@ -192,7 +194,7 @@ public class ParamsFileWriterTests
 }")]
         public void params_file_with_no_errors_should_compile_correctly(string paramsText, string jsonText)
         {           
-          var model = new ParamsSemanticModel(SourceFileFactory.CreateBicepParamFile(PathHelper.FilePathToFileUrl("parameters.bicepparam"), paramsText), BicepTestConstants.FileResolver);
+          var model = new ParamsSemanticModel(SourceFileFactory.CreateBicepParamFile(PathHelper.FilePathToFileUrl("parameters.bicepparam"), paramsText), new ImmutableArray<IDiagnostic>());
```

Comment:
I need to pass empty diagnostics to some of theses tests, should I add this to ```BicepTestConstants```?